code/datums/religions.dm

```diff
@@ -715,10 +757,53 @@
 	male_adept = "Nanotrasen Officer"
 	female_adept = "Nanotrasen Officer"
 	keys = list("security", "space law", "law", "nanotrasen", "centcomm")
+	convert_method = "performing a ritual with a flashbang and a screwdriver. You need to hold the flashbang, with its timer set to 5 seconds, your convert needs to hold the screwdriver and have a free empty hand."
+
 
 /datum/religion/security/equip_chaplain(var/mob/living/carbon/human/H)
 	H.equip_or_collect(new /obj/item/clothing/head/centhat(H), slot_head)
 
+/datum/religion/security/convertCeremony(var/mob/living/preacher, var/mob/living/subject)
+	var/held_banger = preacher.find_held_item_by_type(/obj/item/weapon/grenade/flashbang)
+	if (!held_banger)
+		to_chat(preacher, "<span class='warning'>You need to hold a flashbang to begin the conversion.</span>")
+		return FALSE
+	var/held_screwdriver = subject.find_held_item_by_type(/obj/item/weapon/screwdriver)
+	if (!held_screwdriver)
+		to_chat(preacher, "<span class='warning'>The subject needs to hold a screwdriver to begin the conversion.</span>")
+		return FALSE
+
+	var/obj/item/weapon/grenade/flashbang/F = preacher.held_items[held_banger]
+	var/obj/item/weapon/screwdriver/S = subject.held_items[held_screwdriver]
+
+	if (F.det_time != 50) // The timer isn't properly set
+		to_chat(preacher, "<span class='warning'>The timer in the flashbang isn't properly set up. Set it to 5 seconds.</span>")
+		return FALSE
+
+	subject.visible_message("<span class='notice'>\The [preacher] attemps to convert \the [subject] to [name].</span>")
+
+	if(!convertCheck(subject))
+		subject.visible_message("<span class='warning'>\The [subject] refuses conversion.</span>")
+		return FALSE
+
+	preacher.u_equip(F)
+
+	// Everything is ok : begin the conversion
+	if (!subject.put_in_hands(F))
+		subject.visible_message("<span class='warning'>\The [subject] accepted conversion, but didn't manage to pick up the flashbang. How embarassing.</span>")
+		return FALSE
+
+	// BANGERBOIS WW@
+	sleep(0.1 SECONDS)
+	F.attackby(S, subject)
+	sleep(0.1 SECONDS)
+	F.attackby(S, subject)
+
+	subject.visible_message("<span class='notice'>\The [subject] masterfully completed the delicate ritual. He's now a full-fledged follower of [deity_name].</span>")
+
+	convert(subject, preacher)
```

Comment:
>doesn't actually set the banger to instant detonation

Reply:
It sets it to one second in the convertCeremony (two screwdriver hits)

Reply:
oh, okay.